docs/cli.md

```diff
@@ -151,6 +151,9 @@ profile_models:
 
 ```
 
+**Note**: The checkpoint directory should be removed between consecutive runs of
+the `model-analyzer profile` command.
```

Comment:
Please add this in the end. "If you use --override-output-model-repository flag, this will be done automatically".

Reply:
added 👍